header.css

```diff
@@ -0,0 +1,131 @@
+.toolbar {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  background-color: #3c3a3c;
+  color: azure;
+}
+
+.main {
+  display: flex;
+  flex-direction: column-reverse;
+  justify-content: center;
+  background-color: #1c1a19;
+  color: azure;
+  padding: 120px 24px 120px 24px;
+  background-image: url("Group.png");
+  background-repeat: no-repeat;
+  background-position: bottom -36px right;
+  max-width: 100%;
+}
+
+.bars > ul {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+  gap: 22px;
+  list-style-type: none;
+  padding-right: 30px;
+}
+
+.logo > a {
+  font-size: 22px;
+  text-decoration: none;
+  padding-left: 20px;
+  list-style-type: none;
+  color: #fff5e1;
+}
+
+.contacts > li > a {
+  color: #dbd8d7;
+}
+
+.bars > ul > li > a {
+  display: none;
+}
+
+.bars > ul > .a {
+  display: flex;
+}
+
+.intro {
+  color: #ff6b00;
+  font-size: 56px;
+  font-family: 'Crete Round', serif;
+  font-weight: 400;
+  letter-spacing: 0.37px;
+  line-height: 64px;
+  margin-bottom: 0;
+}
+
+.title {
+  color: #fff5e1;
+  font-size: 24px;
+  font-family: 'Roboto', sans-serif;
+  line-height: 32px;
+  font-style: normal;
+  font-weight: 700;
+  margin-bottom: 0;
+}
+
+.text {
+  font-size: 16px;
+  font-family: 'Poppins', sans-serif;
+  font-weight: 400;
+  font-style: normal;
+  line-height: 24px;
+}
+
+.contacts {
+  display: flex;
+  flex-direction: row;
+  align-items: flex-start;
+  gap: 16px;
+  padding-left: 20px;
+}
+
+.contacts > li {
+  list-style-type: none;
+}
+
+@media screen and (min-width: 992px) {
+  .toolbar {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    background-color: #3c3a3c;
+    color: azure;
+    width: 100vw;
+  }
+
+  .bars > ul > li > a {
+    display: flex;
+    justify-content: center;
+    text-decoration: none;
+    color: #fff5e1;
+  }
+
+  .bars > ul > .a {
+    display: none;
+  }
+
+  .main {
+    display: grid;
+    grid-template-columns: auto auto;
+    gap: 100px;
+    color: azure;
+    padding-right: 50%;
+    background-image: url("Group-all.png");
+    background-repeat: no-repeat;
+    background-position: bottom right;
+  }
+
+  .unordered > .contacts {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-around;
+    gap: 22px;
+    margin-top: 100px;
+    color: #fff5e1;
+  }
+}
```

Comment:
- [ ] Please, try to fix the gap that you have on the left of your portfolio's desktop version: 
<img width="509" alt="image" src="https://user-images.githubusercontent.com/77190203/150616588-326aa4e3-c7d1-4fe2-b0f0-dea6550aca04.png">